src/particles/distribution/T2Z.H

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2021 Chad Mitchell, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_T2Z_H
+#define IMPACTX_T2Z_H
+
+#include "particles/ImpactXParticleContainer.H"
+
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+namespace distribution
+{
+    struct T2Z
+    {
+        using PType = ImpactXParticleContainer::ParticleType;
+
+        /** A T-to-Z transformation
+         *
+         * @param pzd: Design value of pz/mc = beta*gamma.
```

Comment:
Doxygen syntax is just `@param name description`:
```suggestion
         * @param pzd Design value of pz/mc = beta*gamma.
```